train.py

```diff
@@ -187,9 +199,6 @@ def loss_fn(pred, labels):
 
     train_state = TrainState()
 
-    metric_logger = build_metric_logger(job_config, parallel_dims)
-    metric_logger.log_hparams(job_config.args_dict)
```

Comment:
This was moved down, it's still there

Reply:
1. it should be before CheckpointManager to pass experiment hash to aim
2. the other implementation is wrong

I believe this happened because of incorrect merge with current main branch